src/newlines.cpp

```diff
@@ -3263,6 +3275,10 @@ void newlines_cleanup_braces(bool first)
       {
          newline_iarf(pc, AV_REMOVE);
       }
+      else if (pc->type == CT_BYREF)
+      {
+         newline_iarf(pc, AV_REMOVE);
+      }
```

Comment:
This inserted block seems to be responsible for the regression described in #1360.